api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -125,4 +146,31 @@ public void deleteUser(String username) {
       throw ExceptionUtils.convertGoogleIOException(e);
     }
   }
+
+  private String randomString(){
+    String random = IntStream.range(0, 17).boxed().
+      map(x -> ALLOWED.charAt(rnd.nextInt(ALLOWED.length()))).
+      map(Object::toString).
+      collect(Collectors.joining(""));
+    return random;
```

Comment:
This could even be
```
return IntStream.range(0, 17).boxed().
      map(x -> ALLOWED.charAt(rnd.nextInt(ALLOWED.length()))).
      map(Object::toString).
      collect(Collectors.joining(""));
```